jupyterhub/apihandlers/users.py

```diff
@@ -135,6 +150,15 @@ async def get(self, name):
         requester = self.current_user
         if requester.admin:
             model['auth_state'] = await user.get_auth_state()
+        self.eventlog.record_event(
+            eventlogging_schema_fqn('user-action'),
+            1,
+            {
+                'action': 'get',
+                'target_user': {'username': user.name, 'admin': user.admin},
```

Comment:
Let's try to use the same fields as the user model, so `'name'` here so it's `user.name` instead of the redundant `user.username`